plugins/rollbar/lib/samson_rollbar/samson_plugin.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+require 'rollbar'
+
+module SamsonRollbar
+  class Engine < Rails::Engine
+    initializer "rollbar.user_informer" do |app|
+      app.config.middleware.insert_before(::Rack::Runtime, SamsonRollbar::RollbarUserInformer)
+    end
+
+    def self.error_sync(exception, options)
+      Rollbar.with_config(use_async: false) { Rollbar.error(exception, options) }
```

Comment:
we don't need to be sync, all we ned is the UUID

Reply:
Don't we want sync to wait for Rollbar to finish reporting before killing the process?

Reply:
no, rollbar waits "at_exit" before letting the process finish

Reply:
we might still get into trouble with our `exec`based restarts, but that's a minor issue ...

Reply:
So if `sync` is passed in, we want to return the url otherwise nil? Or do we always want to return the url?